book/website/reproducible-research/compendia.md

```diff
@@ -21,21 +21,21 @@ The main purpose is that all elements of a project are published together, so a
 
 ![Cartoon of a research compendium](../../figures/ResearchCompendium.jpg)
 
-## How this will help you/why this is useful
+## How This Will Help You/Why This Is Useful
 
 A research compendium combines all elements of your project, allowing others to reproduce your work, and should be the final product of your research project.
 Publishing your research paper along with a research compendium allows others to access your input, test your analysis, and, if the compendium can be executed, rerun to assess the resulting output.
 This does not only instill trust in your research but can give you more visibility.
 Others may use your research in unexpected ways, some of which are discussed below (refer section: [Using a research compendium](#using-a-research-compendium).
 
-## Chapter content
+## Chapter Content
```

Comment:
This should be updated to 'Summary' as per the [new template](https://github.com/alan-turing-institute/the-turing-way/blob/master/book/templates/chapter-template/chapter-landing-page.md).